src/kw_config_loader.sh

```diff
@@ -141,9 +141,21 @@ function parse_configuration()
 # higher level setting definitions to overwrite lower level ones.
 function load_configuration()
 {
-  parse_configuration "$KW_ETC_DIR/$CONFIG_FILENAME"
-  parse_configuration "$HOME/.kw/$CONFIG_FILENAME"
-  parse_configuration "$PWD/$CONFIG_FILENAME"
+  local -a config_dirs
+  local config_home
+
+  config_home="${XDG_CONFIG_HOME:-"$HOME/.config"}/kw"
+  IFS=':'
+  read -r -a config_dirs <<< "${XDG_CONFIG_DIRS:-"/etc/xdg"}"
+  unset IFS
+
+  for folder in "$PWD" "$config_home" "${config_dirs[@]/%/"/kw"}" \
+    "$KW_ETC_DIR"; do
+    if [[ -f "$folder/$CONFIG_FILENAME" ]]; then
+      parse_configuration "$folder/$CONFIG_FILENAME"
+      return
+    fi
+  done
```

Comment:
Why not use a simple set of `parse_configuration` as we had before. Maybe I did not get the idea, but this looks a little bit overkill to me.

Reply:
You're right, I confused some things. I thought only one file had to be read, but actually all of them must be read. I'll change back to the previous behavior

Reply:
You're right, I confused some things. I thought only one file had to be read, but actually all of them must be read. I'll change back to the previous behavior